### DISCUSSION TITLE
Should there be a `--quiet` option for `bundle add`?

### DISCUSSION THREAD

**Discussion by jeromedalbert** [ruby/rubygems#8154] [Ideas]
`bundle install` has a `--quiet` option to not print progress information to stdout.

Similarly, should there be a `--quiet` option for `bundle add`?

I am asking because a couple weeks ago I fixed an issue in Rails where they were doing `bundle add --quiet`, which would error out since it wasn't a valid option: https://github.com/rails/rails/pull/53083/files. This made me wonder if such an option should be added to `bundle add`.

**Comment by deivid-rodriguez** [comment]
I think it'd make sense, yes.

**Reply by jeromedalbert** [reply]
Ok, I opened a PR: https://github.com/rubygems/rubygems/pull/8157

**Comment by deivid-rodriguez** [comment]
This was implemented through #8157.
